CHANGELOG.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [v2.5.4] - 2020-05-01
+
+### Fixed
+
+- Resolved deprecation warnings on ruby 2.7
+
+### Changed
+
+- Run CI tests against more ruby versions (added 2.6 and 2.7)
```

Comment:
No biggie but I wonder if this is changelog-worthy. Do we consider this stating that those rubies are officially supported from now on? Otherwise there's no impacts on consumers of this gem.

Reply:
Yeah I had that question too. I figured we should be striving to support newer versions as appropriate and keep on top of things where we can

Reply:
Fair enough!